keps/sig-windows/0000-20181221-windows-group-managed-service-accounts-for-container-identity.md

```diff
@@ -99,105 +101,167 @@ A developer has a Windows web app that they would like to deploy in a container
 
 When this app moves to production on containers, the team will need to coordinate to make sure the right GMSA is carried over to the container deployment.
 
-1. Active Directory domain administrator will:
+1. An Active Directory domain administrator will:
 
   - Join all Windows Kubernetes nodes to the Active Directory domain.
   - Provision the GMSA and gives details to application admin.
```

Comment:
specifically, it is not the application admin that needs the GMSA details, but the Kubernetes cluster admin. `- Provision the GMSA and gives details to application admin.`

the application admin/developer will only need the artifact that is produced by the kubernetes cluster admin

Reply:
Agree .. will change.